Xamarin.Essentials/Contacts/Contact.android.cs

```diff
@@ -0,0 +1,161 @@
+            try
+            {
+                var contactPicker = new Intent(Activity, typeof(ContactActivity));
+                contactPicker.SetFlags(ActivityFlags.NewTask);
+                Activity.StartActivity(contactPicker);
+
+                CallBack = (phoneContact) =>
+                {
+                    var tcs = Interlocked.Exchange(ref source, null);
+                    tcs?.SetResult(phoneContact);
+                };
+            }
+            catch (Exception ex)
+            {
+                source.SetException(ex);
+            }
+
+            return source.Task;
+        }
+
+        internal static PhoneContact PlatformGetContacts(Net.Uri contactUri)
+        {
+            using var context = Activity.ContentResolver;
+
+            using var cur = context.Query(contactUri, null, null, null, null);
+            var emails = new Dictionary<string, ContactType>();
+            var phones = new Dictionary<string, ContactType>();
+            var bDate = string.Empty;
+
+            if (cur.MoveToFirst())
+            {
+                var name = cur.GetString(cur.GetColumnIndex(ContactsContract.Contacts.InterfaceConsts.DisplayName));
+                string id;
+
+                id = cur.GetString(cur.GetColumnIndex(ContactsContract.CommonDataKinds.Email.InterfaceConsts.ContactId));
+
+                var idQ = new string[] { id };
+
+                var projection = new string[]
+                {
+                        ContactsContract.CommonDataKinds.Phone.Number,
+                        ContactsContract.CommonDataKinds.Phone.InterfaceConsts.Type,
+                };
+
+                var cursor = context.Query(
+                   ContactsContract.CommonDataKinds.Phone.ContentUri,
+                   null,
+                   ContactsContract.CommonDataKinds.Phone.InterfaceConsts.ContactId + " = ?",
+                   idQ,
+                   null);
+
+                if (cursor.MoveToFirst())
+                {
+                    do
+                    {
+                        var phone = cursor.GetString(cursor.GetColumnIndex(projection[0]));
+                        var phoneType = cursor.GetString(cursor.GetColumnIndex(projection[1]));
+
+                        var contactType = GetContactType(phoneType);
+                        if (!phones.ContainsKey(phone))
+                            phones.Add(phone, contactType);
+                    }
+                    while (cursor.MoveToNext());
+                }
+                cursor.Close();
+
+                projection = new string[]
+                {
+                        ContactsContract.CommonDataKinds.Email.Address,
+                        ContactsContract.CommonDataKinds.Email.InterfaceConsts.Type
+                };
+
+                cursor = context.Query(ContactsContract.CommonDataKinds.Email.ContentUri, null, ContactsContract.CommonDataKinds.Email.InterfaceConsts.ContactId + " = ?", idQ, null);
```

Comment:
should there be a space in = ?

Reply:
I did some local tests, and just the `AND` needs the space before and after.